navigation.yaml

```
@@ -328,6 +328,9 @@ download:
           path: /download/iot/up-squared-iot-grove-server
           hidden: True
 
+         - title: Installation media
+          path: /download/iot/installation-media
```

You probably also want `hidden: True` like all the others:

```suggestion
           path: /download/iot/installation-media
           hidden: True
```